src/widgets/trOverview.test.js

```diff
@@ -0,0 +1,259 @@
+import { TRAINING_REPORT_STATUSES } from '@ttahub/common';
+import db, {
+  EventReportPilot,
+  SessionReportPilot,
+  Recipient,
+  Grant,
+  User,
+} from '../models';
+import {
+  createUser,
+  createGrant,
+  createRecipient,
+  createSessionReport,
+  createTrainingReport,
+} from '../testUtils';
+import trOverview from './trOverview';
+
+// We need to mock this so that we don't try to send emails or otherwise engage the queue
+jest.mock('bull');
+
+describe('TR overview widget', () => {
+  let userCreator;
+  let userPoc;
+  let userCollaborator;
+
+  let recipient1;
+  let recipient2;
+  let recipient3;
+  let recipient4;
+  let recipient5;
+
+  let grant1;
+  let grant2;
+  let grant3;
+  let grant4;
+  let grant5;
+
+  let trainingReport1;
+  let trainingReport2;
+  let trainingReport3;
+
+  beforeAll(async () => {
+    // user/creator
+    userCreator = await createUser();
+    // user/poc
+    userPoc = await createUser();
+    // user/collaborator ID
+    userCollaborator = await createUser();
+
+    // recipient 1
+    recipient1 = await createRecipient();
+    // recipient 2
+    recipient2 = await createRecipient();
+    // recipient 3
+    recipient3 = await createRecipient();
+    // recipient 4
+    recipient4 = await createRecipient();
+    // recipient 5 (only on uncompleted report)
+    recipient5 = await createRecipient();
+
+    // grant 1
+    grant1 = await createGrant({ recipientId: recipient1.id, regionId: userCreator.homeRegionId });
+    // grant 2
+    grant2 = await createGrant({ recipientId: recipient2.id, regionId: userCreator.homeRegionId });
+    // grant 3
+    grant3 = await createGrant({ recipientId: recipient3.id, regionId: userCreator.homeRegionId });
+    // grant 4
+    grant4 = await createGrant({ recipientId: recipient4.id, regionId: userCreator.homeRegionId });
+    // grant 5 (only on uncompleted report)
+    grant5 = await createGrant({ recipientId: recipient5.id, regionId: userCreator.homeRegionId });
+
+    // training report 1
+    trainingReport1 = await createTrainingReport({
+      collaboratorIds: [userCollaborator.id],
+      pocIds: [userPoc.id],
+      ownerId: userCreator.id,
+    });
+
+    // - session report 1
+    await createSessionReport({
+      eventId: trainingReport1.id,
+      data: {
+        deliveryMethod: 'in-person',
+        duration: 1,
+        recipients: [{ value: grant1.id }, { value: grant2.id }],
+        numberOfParticipantsVirtually: 0,
+        numberOfParticipantsInPerson: 0,
```

Comment:
Do we count 'numberOfParticipiantsInPerson'? If so not sure if you want to include it in the test data.